fades/envbuilder.py

```diff
@@ -72,14 +73,19 @@ def create_with_virtualenv(self, interpreter):
             logger.exception("Error creating virtualenv:  %s", error)
             exit()
 
-    def create_env(self, interpreter, is_current):
+    def create_env(self, interpreter, is_current, options):
         """Create the virtualenv and return its info."""
         if is_current:
-            logger.debug("Creating virtualenv with pyvenv")
+            # apply pyvenv options
+            pyvenv_options = options['pyvenv_options']
+            if "--system-site-packages" in pyvenv_options:
+                self.system_site_packages = True
```

Comment:
Shouldn't we remove this option from pyvenv_options? I mean, if we leave it, won't we "apply" it twice?